SCClassLibrary/Common/Control/ServerStatus.sc

```diff
@@ -48,7 +48,8 @@ ServerStatusWatcher {
 		this.prSendNotifyRequest(flag, false);
 	}
 
-	doWhenBooted { |onComplete, limit = 100, onFailure|
+
+	doWhenBooted { |onComplete, limit = 100, onFailure, clock = \AppClock|
 		var mBootNotifyFirst = bootNotifyFirst, postError = true;
```

Comment:
The clock may be an instance, say of a TempoClock. Then `asClass` will be wrong.
Better solution would be to make the default case `nil` and ctach it in `doWhenBooted`, by writing in line 55: 
```
clock = clock ?? { AppClock };
```

Reply:
Thanks! Done in https://github.com/supercollider/supercollider/pull/6198/commits/e30d720f58b59009340438afd158a2b8f94f44d3